runansible

```diff
@@ -33,6 +33,11 @@ then
  exit
 fi
 
+if [ ! -f /etc/ansible/facts.d/local_facts.fact ]; then
+   mkdir -p /etc/ansible/facts.d
+   cp ./scripts/local_facts.fact /etc/ansible/facts.d/local_facts.fact
+fi
+
```

Comment:
I would have the cp ./scripts part after the fi to insure the latest version local_facts.fact is always being used.